app/src/main/java/com/example/movies/ui/settings/SettingsViewModel.kt

```diff
@@ -30,14 +31,23 @@ class SettingsViewModel @Inject constructor(
             val logoutRequestBodyModel = LogoutRequestBodyModel(sessionIdDataCache.loadSessionId())
```

Comment:
It is probably better to put all this logic into LogoutUseCase.

It could be used in the places like this:
com.example.movies.ui.move_details.MovieDetailsFragment#checkMovieAccountState

Reply:
I do not have access to the CookieManager.getInstance() and to the cache in the LogoutUseCase

Reply:
Fixed